templates/docs/base/separators.md

```diff
@@ -20,6 +20,8 @@ View example of the horizontal line
 
 #### Muted horizontal rule
 
+<span class="p-label--new">New</span>
```

Comment:
Should we add New to the Base/Separators in the side navigation?

Reply:
Yes, aded to typography too.